package.json

```diff
@@ -54,6 +53,8 @@
     "css-loader": "^0.28.4",
     "extract-text-webpack-plugin": "^3.0.0",
     "file-loader": "^0.11.2",
+    "flow-bin": "^0.56.0",
```

Comment:
Why you moved flow-bin and flow-typed from dev dependencies?

Reply:
Because ESLint was telling me so :banana:. I've updated the configuration.